packages/twenty-front/src/modules/object-metadata/components/ObjectMetadataItemsRelationPickerEffect.tsx

```diff
@@ -4,6 +4,54 @@ import { useRelationPicker } from '@/object-record/relation-picker/hooks/useRela
 import { IdentifiersMapper } from '@/object-record/relation-picker/types/IdentifiersMapper';
 import { getLogoUrlFromDomainName } from '~/utils';
 
+export const identifierMapper: IdentifiersMapper = (
```

Comment:
I did not modified the function. I just moved it out so I could export it in the test